sheetstorm/tests/test_unit_generate_test_data.py

```diff
@@ -0,0 +1,207 @@
+        self, user_type, number_of_users_in_database, number_of_users_to_create
+    ):
+        Command().create_user(user_type, number_of_users_in_database)
+        Command().create_user(user_type, number_of_users_to_create)
+        self.assertEqual(CustomUser.objects.filter(user_type=user_type).count(), number_of_users_in_database)
+
+
+class CreateProjectMethodsTests(TestCase):
+    def setUp(self):
+        self.options = {ProjectType.SUSPENDED.name: 4, ProjectType.ACTIVE.name: 5, ProjectType.COMPLETED.name: 7}
+
+        for _project_number in range(2):
+            ProjectFactory(suspended=True)
+
+        for _project_number in range(2):
+            ProjectFactory()
+
+        for _project_number in range(2):
+            ProjectFactory(stop_date=timezone.now() - relativedelta(days=1))
+
+        self.number_of_suspended_projects_to_create = (
+            self.options.get(ProjectType.SUSPENDED.name) - Project.objects.filter_suspended().count()
+        )
+        self.number_of_active_projects_to_create = (
+            self.options.get(ProjectType.ACTIVE.name) - Project.objects.filter_active().count()
+        )
+        self.number_of_completed_projects_to_create = (
+            self.options.get(ProjectType.COMPLETED.name) - Project.objects.filter_completed().count()
+        )
+
+    def tearDown(self):
+        Project.objects.all().delete()
+
+    @parameterized.expand(
+        [(ProjectType.SUSPENDED.name, 2), (ProjectType.ACTIVE.name, 3), (ProjectType.COMPLETED.name, 5)]
+    )
+    def test_number_of_projects_to_create_should_be_a_difference_of_number_of_projects_in_database_and_definite_number(
+        self, project_type, expected_result
+    ):
+        self.assertEqual(Command._compute_number_of_projects_to_create(self.options, project_type), expected_result)
+
+    def test_compute_number_of_projects_to_create_function_should_raise_error_if_project_type_does_not_exist(self):
+        with self.assertRaises(UnsupportedProjectTypeException):
+            Command._compute_number_of_projects_to_create(self.options, "error")
+
+    @parameterized.expand([ProjectType.SUSPENDED.name, ProjectType.ACTIVE.name, ProjectType.COMPLETED.name])
+    def test_compute_number_of_projects_to_create_function_should_return_0_if_number_of_projects_is_not_specified(
+        self, project_type
+    ):
+        self.assertEqual(Command._compute_number_of_projects_to_create({project_type: None}, project_type), 0)
+
+    def test_number_of_projects_to_create_should_be_set_and_equal_to_computed_number(self):
+        expected_options = {
+            ProjectType.ACTIVE.name: self.number_of_active_projects_to_create,
+            ProjectType.SUSPENDED.name: self.number_of_suspended_projects_to_create,
+            ProjectType.COMPLETED.name: self.number_of_completed_projects_to_create,
+        }
+        self.assertDictEqual(Command(self.options)._set_number_of_projects_to_create(self.options), expected_options)
```

Comment:
It would be enough to use just `self.assertEqual` instead of `self.assertDictEqual`

Reply:
Done